doc/user/content/concepts/sinks.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Sinks
+description: Conceptual page on sinks.
+menu:
+  main:
+    parent: concepts
+    weight: 25
+    identifier: 'concepts-sinks'
+---
+
+## Overview
```

Comment:
Same content as before (even left the erroneous "only supports" kafka statement) -- just to keep the reviews digestible.